### DISCUSSION TITLE
How to find the wrong image data in the dataset?

### DISCUSSION THREAD

**Discussion by shenao1995** [Project-MONAI/tutorials#957] [Q&A]
I encountered a problem while doing a segmentation task and the following error was reported: `TypeError: new(): invalid data type 'str'`. This should be a problem with an image data, but how do I find this problematic data?  Please help me to solve this problem. Thanks! 
![微信截图_20220923213656](https://user-images.githubusercontent.com/28835780/191973200-0ee970ed-867a-43a2-b165-273f7ba74af2.png)

**Comment by wyli** [comment]
related to this https://github.com/Project-MONAI/MONAI/discussions/5167 cc @mingxin-zheng @Nic-Ma
